### DISCUSSION TITLE
mse: streams: wrong response on describe

### DISCUSSION THREAD

**Discussion by manjump** [fuatakgun/eufy_security#1152] [Q&A]
HI,
I have battery cam eufy. Obviously, the duration of the stream is limited to save battery. On the application, a message warns us to stop viewing. With RTSP, when the stream cuts out, an error message (mse: streams: wrong response on describe) appears with a black screen. Is there a solution to get around the problem?

**Comment by HarryBouch96** [comment]
It seems many people are experiencing this issue. I have found a work around and that is to create an automation which starts the RTSP stream every 5 minutes using the entity provided. This means that I can no longer stop the stream though or else it just starts again after 5 minutes but a continuous stream is fine for my needs.
